app/src/pages/inside/logsPage/logsGrid/logsGrid.jsx

```diff
@@ -156,27 +139,53 @@ export class LogsGrid extends Component {
       },
       sortable: true,
       component: () => <div />,
-    },
-    {
-      id: TIME_COLUMN_ID,
-      title: {
-        full: this.props.intl.formatMessage(messages.timeColumnTitle),
+    };
+    const columns = [
+      {
+        id: 'logMessage',
+        title: {
+          component: LogMessageSearch,
+          componentProps: {
+            filter: this.props.filter,
+            onFilterChange: this.props.onFilterChange,
+          },
+        },
+        sortable: true,
+        maxHeight: 200,
+        component: MessageColumn,
+        customProps: {
+          markdownMode: this.props.markdownMode,
```

Comment:
Also, please add
`consoleView: this.props.consoleView`, otherwise it's not passed anywhere